### ISSUE TITLE
Installing plugins on windows?

### ISSUE THREAD

**Issue by KeePach** [comictagger/comictagger#679]
Hi, I'm using windows 10 and the latest a.23 windows exe.

I found out you can install plugins for other databases. I downloaded the latest alpha version and figuring with the logs where are you're supposed to install the plugins I tried to use the mangadex one, I downloaded the zip master file and placed the folder where the logs said the plugin folder was, when launching comictagger, this error is shown on the logs.
```
2024-09-08T12:04:05 | comictalker | ERROR | Failed to load talker: mangadex
Traceback (most recent call last):
  File "comictalker\__init__.py", line 36, in get_talkers
  File "importlib\metadata.py", line 86, in load
  File "importlib\__init__.py", line 127, in import_module
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "C:\Users\jhmp_\AppData\Local\ComicTagger\ComicTagger\mangadex_talker-master\mangadex\mangadex.py", line 166, in <module>
    class MangaDexResponse(TypedDict, Generic[T], total=False):
  File "typing.py", line 1947, in __new__
TypeError: cannot inherit from both a TypedDict type and a non-TypedDict base class
```

everything else was fine and the app launched without the mangadex talker. hope someone can help me. thanks.

**Comment by lordwelch** [comment]
Plugins are still very much experimental so thanks for trying it out, you will need to open an issue here https://github.com/mizaki/mangadex_talker as it looks like an issue with the mangadex code and @mizaki should be able to help you with it

**Comment by KeePach** [comment]
I'll do that, thanks!

**Comment by mizaki** [comment]
This has been resolved with aec9f5f1333cb0930b1fa60ea76d5035f37656a0 For reference `TypedDict` got broken around 3.8 in `typing`, using `typing_extensions` is a workaround but `typing` has been fixed in 3.11.
